### PR TITLE
Creates the view controller files and view model files for all the views

### PR DESCRIPTION
**Pull request by MatthewCHill**

### CODE REVIEW

File: ConnectionJustForToday/ConnectionJustForToday/Models/SPADReading.swift [hunk 1]
```diff
@@ -15,13 +15,13 @@ class SPADReading {
     let title: String
     let quote: String
     let body: String
-    let aspiration: String
+    let affirmation: String
     
     init(date: String, title: String, quote: String, body: String, aspiration: String) {
```
Comment (maxpoff) [inline ConnectionJustForToday/ConnectionJustForToday/Models/SPADReading.swift]: You have two different names here. Is that intentional? You should pick either affirmation or aspiration and keep it consistent

Comment (MatthewCHill) [reply ConnectionJustForToday/ConnectionJustForToday/Models/SPADReading.swift]: It wasn't. Ill be refactoring to Affirmation.

Comment (maxpoff) [reply ConnectionJustForToday/ConnectionJustForToday/Models/SPADReading.swift]: Once you fix that, I will approve and you can merge

Comment (MatthewCHill) [reply ConnectionJustForToday/ConnectionJustForToday/Models/SPADReading.swift]: Resolved. Also fixed my GoogleService-Info on this branch. The Bundle-ID was incorrect.

**Review by maxpoff — APPROVED** [review]

**Pull request merged**